src/components/ReadiumView.web.tsx

```diff
@@ -38,6 +38,14 @@ export const ReadiumView = React.forwardRef<{
     },
   }));
 
+  // There can be a created field on the location here,
+  // and it will cause an infinite loop if not eliminated
+  // @ts-ignore
+  if (location?.created) {
```

Comment:
I've never seen an infinite loop occurring here, can you specify the sequence of events that causes this loop to occur?

Rather than mutating the location property, is it possible to pass a primitive attribute of `location` into [the `useDeepCompareEffect` deps](https://github.com/5-stones/react-native-readium/blob/main/web/hooks/useLocationObserver.ts#L16)? For instance, I believe there's a specific attributes that we could observe on `Locator` (yes I realize now that we're calling a `Locator` type object `location`, which was my mistake), like `location.href`, `location.type`, & `location.locations`. I believe these properties sufficiently represent the unique location, that we'd care to perform a `goTo` on.

Reply:
I noticed the infinite loop when I put a console.log inside of the useDeepCompareEffect whenever I was reading a book. It seemed like the reader.goTo must make a new location somewhere, which includes the created time. That was triggering the dependencies since the created time changed, which then would call the reader.goTo again, making a new created time.

Yeah, just changing the dependencies would probably work. It is odd that there is a created field on that object since that isn't supposed to be in the locator type, but maybe the d-i-t-a reader has a slightly different type definition.